eng/pipelines/common/xplat-setup.yml

```diff
@@ -88,26 +88,22 @@ jobs:
     ${{ if eq(parameters.jobParameters.pool, '') }}:
       pool:
         # Public Linux Build Pool
-        ${{ if and(in(parameters.osGroup, 'Linux', 'FreeBSD'), eq(variables['System.TeamProject'], 'public')) }}:
+        ${{ if and(in(parameters.osGroup, 'Android', 'Linux', 'FreeBSD'), eq(variables['System.TeamProject'], 'public')) }}:
           name:  NetCorePublic-Pool
           queue: BuildPool.Ubuntu.1604.Amd64.Open
 
         # Official Build Linux Pool
-        ${{ if and(in(parameters.osGroup, 'Linux', 'FreeBSD'), ne(variables['System.TeamProject'], 'public')) }}:
+        ${{ if and(in(parameters.osGroup, 'Android', 'Linux', 'FreeBSD'), ne(variables['System.TeamProject'], 'public')) }}:
           name: NetCoreInternal-Pool
           queue: BuildPool.Ubuntu.1604.Amd64
 
         # Public OSX Build Pool
-        ${{ if eq(parameters.osGroup, 'OSX') }}:
-          vmImage: 'macOS-10.14'
+        ${{ if and(in(parameters.osGroup, 'OSX', 'iOS', 'tvOS'), eq(variables['System.TeamProject'], 'public')) }}:
```

Comment:
Why do we want to condition based on teamproject if both vmImages are the same?

Reply:
@safern where are we usually building non-public OSX? Still on hosted pools or our on-prem `OSX.101x.Amd64`?

Reply:
Honestly, that was a placeholder in case that detail changed. I tried using a queue name directly and had issues with it. Pretty sure it does the right thing with the vmImage specification here